packages/@sanity/components/src/buttons/DropDownButton.js

```diff
@@ -164,18 +140,18 @@ class DropDownButton extends React.PureComponent {
                   {isActive => (
```

Comment:
`isActive` is not in use, so you can remove the `<Stacked>`-wrapper here

Reply:
Turns out i need it. Will use the isActive prop